### PR TITLE
Fixing tests

### PR DESCRIPTION
**Pull request by mdbenjam**
### Description
Fixes four flakey tests, skips two, and tries to fix the setup on a bunch of Reader tests. I think this is *most* of our instability.

I've written comments throughout the PR about why I did certain things.

### Testing Plan
1. Circle passes

### CODE REVIEW

File: lib/generators/vacols/case_hearing.rb [hunk 1]
```diff
@@ -1,13 +1,8 @@
 class Generators::Vacols::CaseHearing
   class << self
-    def generate_pkseq
```
Comment (mdbenjam) [inline lib/generators/vacols/case_hearing.rb]: I think the DB generates this for us. For some reason this seems to be causing occasional test problems.

File: spec/controllers/idt/api/appeals_controller_spec.rb [hunk 2]
```diff
@@ -103,7 +103,10 @@
             get :list
             expect(response.status).to eq 200
             response_body = JSON.parse(response.body)["data"]
-            ama_appeals = response_body.select { |appeal| appeal["type"] == "appeals" }
+            ama_appeals = response_body
+              .select { |appeal| appeal["type"] == "appeals" }
+              .sort_by { |appeal| appeal["attributes"]["file_number"]}
```
Comment (mdbenjam) [inline spec/controllers/idt/api/appeals_controller_spec.rb]: About 5% of the time the order of the appeals returned is switched. So we sort before comparing.

File: spec/feature/queue/checkout_flows_spec.rb [hunk 3]
```diff
@@ -401,22 +403,27 @@ def select_issue_level_options(opts)
         delete_btn = find("button", text: "Delete Issue")
         expect(delete_btn.disabled?).to eq true
 
-        fields = page.find_all ".Select--single"
```
Comment (mdbenjam) [inline spec/feature/queue/checkout_flows_spec.rb]: This way of controlling the test is very slow, and not reliable.

File: spec/feature/queue/checkout_flows_spec.rb [hunk 4]
```diff
@@ -605,6 +612,8 @@ def select_issue_level_options(opts)
       vet_name = appeal.veteran_full_name
       click_on "#{vet_name.split(' ').first} #{vet_name.split(' ').last} (#{appeal.sanitized_vbms_id})"
 
+      expect(page).to have_content("Actions")
```
Comment (mdbenjam) [inline spec/feature/queue/checkout_flows_spec.rb]: Check for content on the page before running the `click_dropdown` command which can run without waiting.

File: spec/feature/queue/checkout_flows_spec.rb [hunk 5]
```diff
@@ -614,10 +623,10 @@ def select_issue_level_options(opts)
       click_dropdown 6
       expect(page).to have_content(COPY::COLOCATED_ACTION_PLACE_CUSTOM_HOLD_COPY)
 
-      hold_duration = [rand(100), 1].min
+      hold_duration = [rand(100), 1].max
```
[the remark this answers is not in the document]
Comment (amprokop) [reply spec/feature/queue/checkout_flows_spec.rb]: omgg 🥇

File: spec/feature/queue/checkout_flows_spec.rb [hunk 6]
```diff
@@ -614,10 +623,10 @@ def select_issue_level_options(opts)
...
       fill_in COPY::COLOCATED_ACTION_PLACE_CUSTOM_HOLD_COPY, with: hold_duration
 
-      instructions = generate_words 50
+      instructions = generate_words 5
```
[the remark this answers is not in the document]
Comment (amprokop) [reply spec/feature/queue/checkout_flows_spec.rb]: 💯

File: spec/feature/queue/checkout_flows_spec.rb (hunk 4 above)
Comment (amprokop) [reply spec/feature/queue/checkout_flows_spec.rb]: 😆

File: spec/feature/queue/checkout_flows_spec.rb (hunk 3 above)
Comment (amprokop) [reply spec/feature/queue/checkout_flows_spec.rb]: 🔪

File: lib/generators/vacols/case_hearing.rb (hunk 1 above)
Comment (amprokop) [reply lib/generators/vacols/case_hearing.rb]: 🔮

**Pull request merged**